pkg/cli/rebuild.go

```diff
@@ -51,10 +52,14 @@ func NewCmdRebuild() *cobra.Command {
 					key.Project.Field(project),
 					key.Directory.Field(dir),
 					key.Version.Field(version),
-					key.DiffCount.Field(count),
+					key.DiffCount.Field(uint32(len(diff.Updates))),
```

Comment:
The segfault you are seeing in the JS tests is from this line.

This won't segfault if `diff.Updates` is `nil` but it will segfault if `diff` (which is a pointer) is `nil`.